checker/src/main/java/org/checkerframework/checker/toy/ToyChecker.java

```diff
@@ -0,0 +1,6 @@
+package org.checkerframework.checker.toy;
+
+import org.checkerframework.common.basetype.BaseTypeChecker;
+
+/** A type-checker plug-in for the Toy type system. */
+public class ToyChecker extends BaseTypeChecker {}
```

Comment:
Rather than calling this "Toy", please give it a name that is more evocative of its purpose, which is defaults to type declarations.